packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -198,9 +206,56 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
                 }
             },
             // eth addresses are valid without checksum but Trezor displays them as checksummed
-            checksum: (value: string) => {
-                if (networkType === 'ethereum' && !checkAddressCheckSum(value)) {
-                    return translationString('RECIPIENT_IS_NOT_VALID');
+            used_checksum: async (value: string) => {
```

Comment:
I don't really get the whole part of this plus you are not returning anything so types are complaining. Can you elaborate what it does and why you even use localStorage etc.

Reply:
Also using `addressInfoContent` seems to not be needed because you can return some error as string and if it matches something then in getInputState you can return info so you don't need to modify input state to void.

Reply:
> I don't really get the whole part of this plus you are not returning anything so types are complaining. Can you elaborate what it does and why you even use localStorage etc.

I will remake the solution, it was my first iteration that worked, but i agree that it doesn't look good. I won't use localStorage either.